packages/app-form-builder/src/admin/components/FormEditor/Tabs/EditTab/Field.tsx

```diff
@@ -29,26 +30,70 @@ const Actions = styled("div")({
     alignItems: "right",
     "> *": {
         flex: "1 100%"
+    },
+    ".switch-wrapper": {
+        display: "flex",
+        alignItems: "center",
+        color: "var(--mdc-theme-text-secondary-on-background)",
+        ".webiny-ui-switch": {
+            margin: "0 16px"
+        }
     }
 });
 
+const StyledDivider = styled("div")({
+    width: 2,
+    margin: 5,
+    backgroundColor: "var(--mdc-theme-on-background)"
+});
+
 interface FieldProps {
     field: FbFormModelField;
     onEdit: (field: FbFormModelField) => void;
     onDelete: (field: FbFormModelField) => void;
 }
 const Field: React.FC<FieldProps> = props => {
     const { field, onEdit, onDelete } = props;
-    const { getFieldPlugin } = useFormEditor();
-
+    const { getFieldPlugin, updateField } = useFormEditor();
     const fieldPlugin = getFieldPlugin({ name: field.name });
+
+    const isRequired = useMemo(() => {
+        return field?.validation?.some(validation => validation.name === "required");
+    }, [field]);
```

Comment:
I'd say `useMemo` usage here is not worth it for a couple of reasons, primarily because of the fact the the `field` reference is probably always different on each render, so the recalculation will happen. And even if the ref did not change, the `[field]` would then produce incorrect results in the UI, since that's not the value we want to use as a caching condition. 

This will suffice:

```ts
const isRequired = field?.validation?.some(validation => validation.name === "required");
```